### DISCUSSION TITLE
Syntax highlighting not working in Notebook

### DISCUSSION THREAD

**Discussion by seelk07** [jupyter/notebook#7399] [Q&A]
Syntax highlighting is not enabled when viewing notebook files (*.ipynb) in Jupyter Notebook (python -m notebook).  Syntax highlighting is enabled when viewing the same files via JupyterLab (python -m jupyterlab).  I tried seeing if I could force the syntax highlighting via Settings but the options are greyed out (in Notebook and JupyterLab) after opening files.  Syntax highlighting was working properly on previous versions.  Thoughts?

**Comment by ai-robert** [comment]
Totally agree, glad you raised this. I came here because Notebooks v7 are really missing a lot of features that worked well in previous versions.

**Comment by jtpio** [comment]
Syntax highlighting should normally work fine.

Would you be able to share a screenshot of the issue?

**Comment by seelk07** [comment]
Please note, things are working fine now.  I'm not sure when it changed but syntax highlighting is now working.  Thanks.
